templates/Settings/personal.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * @copyright Copyright (c) 2018 Arthur Schiwon <blizzz@arthur-schiwon.de>
+ *
+ * @author Arthur Schiwon <blizzz@arthur-schiwon.de>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+style('data_request', 'style');
+script('data_request', ['init', 'app']);
+
+
+?>
+
+<div id="data-request" class="section">
+	<h2><?php p($l->t('Account')); ?></h2>
+	<p><?php p($l->t('You can request an export of your data or account deletion from the system administrator. This can take up to 30 days to be executed.')); ?></p>
```

Comment:
Cut the "to be executed", too technical language.

Also, add the class "settings-hint" to the p, just like it is in the other explanatory sentences in settings:
https://github.com/nextcloud/server/blob/master/settings/templates/settings/personal/security.php#L36